src/resources/FundraisingPages.js

```diff
@@ -8,7 +8,7 @@ class FundraisingPages extends Resource {
         'activity',
         'posts'
       ],
-      creates: ['posts'],
+      creates: ['posts', 'transfers'],
```

Comment:
I don't think 'transfers' will work here unless the endpoint is /fundraising-teams/#/transfers.

The only plural conversion we're doing is on the method itself, so `classy.fundraisingTeams.createTransfer(#, {})` would still hit the /fundraising-teams/#/transfers endpoint. We should probably make a custom method for this one.